### PR TITLE
SDK-15 added devicetrustbydevice endpoint and bumped up to version 4.0.0

### PR DESCRIPTION
**Pull request by gadulin**

### CODE REVIEW

File: kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java [hunk 1]
```diff
@@ -535,6 +623,7 @@ CloseableHttpClient getHttpClient() {
 	 * @throws AccessException
 	 *             Thrown if any of the param values are invalid or there was a problem getting a response.
 	 */
+	@Deprecated
```
Comment (boyankukushev) [inline kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: I don't think this method is deprecated, as well as the one below

File: kount-access-java-sdk/src/test/java/com/kount/kountaccess/AccessSDKTest.java [hunk 2]
```diff
@@ -4,16 +4,24 @@
 package com.kount.kountaccess;
```
Comment (boyankukushev) [inline kount-access-java-sdk/src/test/java/com/kount/kountaccess/AccessSDKTest.java]: Let's try and split this big test file to smaller files -- each testing a single endpoint.

File: kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java (hunk 1 above)
Comment (gadulin) [reply kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: It has @deprecated in the java doc and the eclipse "autocompleted" it.

Comment (gadulin) [reply kount-access-java-sdk/src/main/java/com/kount/kountaccess/AccessSdk.java]: Done.

File: kount-access-java-sdk/src/test/java/com/kount/kountaccess/AccessSDKTest.java (hunk 2 above)
Comment (gadulin) [reply kount-access-java-sdk/src/test/java/com/kount/kountaccess/AccessSDKTest.java]: Done.

Comment (gjd-keynetics) [reply kount-access-java-sdk/src/test/java/com/kount/kountaccess/AccessSDKTest.java]: +1

**Pull request merged**